engine/worker/builtin_coverage.go

```diff
@@ -70,6 +85,18 @@ func runParseCoverageResultAction(w *currentWorker) BuiltInAction {
 			return res
 		}
 
+		if min_req > 0 {
+			total = report.TotalLines
+			parsed = report.CoveredLines
+			cov_percent := float(parsed) / total * 100
```

Comment:
it won't compile.

You haven't declared and initialise `total` and `parsed`. I suggest you to delete `total` and `parsed` and just `cov_percent := (float64(report.CoveredLines) / float64(report.TotalLines)) * 100`

Reply:
Oups, sorry wrong refactoring before pushing the patch.